xknx/telegram/apci.py

```diff
@@ -972,6 +979,73 @@ def __str__(self) -> str:
         return "<Restart />"
 
 
+class ConfirmedRestart(APCI):
+    """Restart service."""
+
+    CODE = APCIService.CONFIRMED_RESTART
+
+    def __init__(self, erase_code: bytes, channel: bytes) -> None:
```

Comment:
It would be handier to use `int` instead of `bytes` for channel and erase_code (or an enum for the latter if applicable).

Maybe set the default value of `channel: int = 0` - for some of the erase codes other values are invalid so I guess it would be a proper default.